pytket/extensions/cutensornet/tnstate/mps.py

```diff
@@ -36,168 +34,17 @@
 
 from pytket.extensions.cutensornet.general import set_logger
 
-# An alias so that `intptr_t` from CuQuantum's API (which is not available in
-# base python) has some meaningful type name.
-Handle = int
-# An alias for the CuPy type used for tensors
-try:
-    Tensor = cp.ndarray
-except NameError:
-    Tensor = Any
+from .general import CuTensorNetHandle, Config, TNState, Tensor
 
 
-class DirectionMPS(Enum):
+class DirMPS(Enum):

```

Comment:
I like explicit names, but completely up to you!

Reply:
It's fine, especially since not exposed. I've seen now this change is all over the place, so I am fine with keeping it like this (it is really your call anyway!).